Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -482,6 +489,10 @@ public NodejsProjectNode(NodejsProjectPackage package)
                 NodejsPackage.Instance.CheckSurveyNews(false);
                 ModulesNode.ReloadHierarchySafe();
 
+#if DEV14
+                TryToAcquireTypings(new[] { "node" });
```

Comment:
What if you open an existing folder that doesn't have the typings installed - will this line get executed first? Or what if you create a new project?

Basically, I'm wondering if there's some set of cases that we can anticipate where there are many packages installed and we could execute tsd just once.